src/config/settings.ts

```diff
@@ -25,7 +25,7 @@ const CURRENT_SETTINGS_VERSION = 2;
 const defaultSettings = {
   version:    CURRENT_SETTINGS_VERSION,
   kubernetes: {
-    version:     'v1.19.11',
+    version:     '',
```

Comment:
Just to confirm: the default k8s version is now never set, correct?  In the backend if it isn't set this branch now uses the most recent version. Note that if the user never explicity chooses a version, and they upgrade to a new version of RD with a new latest version, the system will quietly migrate to that newest version. If this is desired behavior, it should be documented somewhere.

If it's not desired, the change to settings should be preserved.

Reply:
That's… a good catch, thanks! We should persist the chosen version, I think.